samza-core/src/main/java/org/apache/samza/clustermanager/ContainerProcessManager.java

```diff
@@ -242,6 +259,8 @@ public void stop() {
 
     // Shutdown allocator thread
     containerAllocator.stop();
+    // Shutdown container placement handler thread
+    containerPlacementHandler.stop();
```

Comment:
Would you need the similar thread join pattern for the container placement handler thread , similar to the container allocator thread?